upcase_booklist_app/upcase_booklist_app.py

```diff
@@ -29,5 +31,28 @@ def authors():
     return render_template("authors.html", books=books)
 
 
+@app.route("/lists")
+def lists():
+    """Organize authors and their books into categories"""
+
+    return render_template("lists.html", categories=categories)
+
+
+@app.route("/lists/<int:category_id>")
+def category(category_id):
+    """Direct user to page about a specific category"""
+
+    selected_cateogry = None
+    category_title = None
+    for category in categories:
+        if category["category_id"] == category_id:
+            selected_cateogry = category["books"]
```

Comment:
Is this the selected category, or the `books` associated with that category? If the latter than your variable nameson this line should be `selected_book. Same on line 53.